Source/Parallelization/WarpXComm_K.H

```
@@ -696,6 +802,7 @@ void warpx_interp (const int j,
 
     else // PSATD (finite-order interpolation)
     {
+#if (AMREX_SPACEDIM > 1) // 1D not implemented for PSATD
```

In the case `AMREX_SPACEDIM = 1`, could you add a runtime error here (e.g. with `amrex::Abort`)?
I am afraid that otherwise we might forget to implement that part of the code if we later implement PSATD 1D, in which case the code might silently skip the interpolation here - which would probably be difficult to detect and debug then.

---

Good idea. I added runtime error with amrex::Abort for PSATD 1D.